pkg/Library/Graph/Atoms/GraphAgent.js

```diff
@@ -17,10 +17,7 @@ async update(inputs, state, {output, isDirty, service}) {
   }
   if (inputs.publishedGraphsUrl && isDirty('publishedGraphsUrl')) {
     state.publishedGraphsUrl = inputs.publishedGraphsUrl;
-    const publicGraphs = await this.loadPublicGraphs(inputs.publishedGraphsUrl);	
-    if (publicGraphs) {
-      output({publicGraphs});
-    }
+    output(await this.loadPublicGraphs(inputs.publishedGraphsUrl));
   }
   const outputs = await this.initGraphs(inputs, state);
```

Comment:
Similar to above, `output` being opaque here forces the reader to spelunk for the values. Suggest unpacking the outputs and returning them explicitly.

Yes, it's harder to maintain, but I think it's worth it for long term survival.

Reply:
reverted this change, it was more readable previously